lib/eval_on_dart_library.dart

```diff
@@ -0,0 +1,57 @@
+import 'dart:async';
+import 'package:vm_service_lib/vm_service_lib.dart';
+
+import 'globals.dart';
+
+class EvalOnDartLibrary {
+  EvalOnDartLibrary(String libraryName, VmService service) {
```

Comment:
idiomatic dart is
EvalOnDartLibrary(this.libraryName, this.service) {

That would make libraryName service public but that isn't a horrible thing.

Reply:
Done.